<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Choose OS (Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="windowsCheckbox">
            <property name="text">
             <string>Windows</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="macCheckbox">
            <property name="text">
             <string>Mac</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="linuxCheckbox">
            <property name="text">
             <string>Linux</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Choose Drink (non exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="beerCheckbox">
            <property name="text">
             <string>Beer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="juiceCheckbox">
            <property name="text">
             <string>Juice</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="coffeeCheckbox">
            <property name="text">
             <string>Coffee</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Choose Answer:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QRadioButton" name="aRadioButton">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bRadioButton">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cRadioButton">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Get Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Set state</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
